tsl/test/expected/cagg_policy.out

```diff
@@ -1400,3 +1400,70 @@ SELECT timescaledb_experimental.add_policies('cagg');
  f
 (1 row)
 
+-- Issue #6902
```

Comment:
Might be good to add a description of the issue here and describe what the test does, even if you refer to the issue. Otherwise a developer need to find the issue to understand what this is testing.